ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml

```diff
@@ -1,11 +1,10 @@
 tag_specification:
-  cluster: https://api.ci.openshift.org
-  name: origin-v3.11
-  namespace: openshift
+  name: '4.0'
+  namespace: ocp
 base_images:
   base:
     cluster: https://api.ci.openshift.org
-    name: origin-v3.11
+    name: '4.0'
     namespace: openshift
```

Comment:
I believe the namespace needs to be `ocp` here as well

Reply:
Please check now